components/ephemeral_storage/browser/BUILD.gn

```diff
@@ -0,0 +1,16 @@
+source_set("browser") {
+  sources = [
+    "ephemeral_storage_tab_helper.cc",
+    "ephemeral_storage_tab_helper.h",
+  ]
+
+  deps = [
+    "//base",
```

Comment:
This list of deps does not appear to correspond to the headers in ephemeral_storage_tab_helper.cc/ephemeral_storage_tab_helper.h, where did you come up with these?

Reply:
Fixed. This was a copy-and-paste issue. Thanks for pointing it out.